### DISCUSSION TITLE
Warehouse records

### DISCUSSION THREAD

**Discussion by toiletman74** [nzp-team/nzportable#230] [Records and Speedruns]
![image](https://user-images.githubusercontent.com/91436591/223645190-2f901e05-bebd-48a2-a2c1-9c649903691c.png)
Don't know if it's a new record or anything but it's probably a good place to start this board. This is my PB.

**Comment by toiletman74** [comment]
New pb poggers
![image](https://user-images.githubusercontent.com/91436591/224876788-9f954f4f-bb0a-43e7-9b4d-7f2b26fb38aa.png)

**Comment by Vinxent9** [comment]
![whc](https://github.com/nzp-team/nzportable/assets/56005655/b48cf70f-427d-483f-b40f-ccae540dfc48)
got messed up

**Comment by cckaulitz-rgb** [comment]
![IMG_3622](https://github.com/user-attachments/assets/c8724805-6a48-423b-bff9-0822925abe21)

**Reply by cckaulitz-rgb** [reply]
My pc still in sleep so I’m still continuing

0 downs🥹✌️
